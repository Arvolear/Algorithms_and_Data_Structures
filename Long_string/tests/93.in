45891923821176262153686
14624960640744346473