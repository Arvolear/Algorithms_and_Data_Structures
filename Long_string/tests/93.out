45906548781817006500159
45877298860535517807213
671167579612740724135456614364459558049478
3137